cartography/intel/azure/sql.py

```diff
@@ -647,169 +633,148 @@ def load_database_details(neo4j_session, details, update_tag):
 
 
 @timeit
-def _load_replication_links(neo4j_session, replication_links, update_tag):
+def _load_replication_links(
+        neo4j_session: neo4j.Session, replication_links: List[Dict], update_tag: int,
+) -> None:
     """
     Ingest replication links into neo4j.
     """
     ingest_replication_links = """
-    MERGE (rl:AzureReplicationLink{id: {LinkId}})
+    UNWIND {replication_links_list} as replication_link
+    MERGE (rl:AzureReplicationLink{id: replication_link.id})
     ON CREATE SET rl.firstseen = timestamp(), rl.lastupdated = {azure_update_tag}
```

Comment:
same as above. move lastupdated to update set. move location to create set

Reply:
done